### PR TITLE
Tmux cheat sheet: Added more shortcuts and fixed typos

### PR DESCRIPTION
**Pull request by mayank7**
Tmux cheat sheet: Added more shortcuts and fixed typos

IA Page: https://duck.co/ia/view/tmux_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Tmux Cheat Sheet](https://duck.co/ia/view/tmux_cheat_sheet)

**Description**: Tmux cheat sheet

**Example Query**: [tmux cheat sheet](https://beta.duckduckgo.com/?q=tmux cheat sheet), [tmux quick reference, tmux guide, tmux cheat sheet, tmux commands](https://beta.duckduckgo.com/?q=tmux quick reference, tmux guide, tmux cheat sheet, tmux commands)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/tmux_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/tmux_cheat_sheet)*

File: share/goodie/cheat_sheets/json/tmux.json [hunk 1]
```diff
@@ -16,15 +16,27 @@
...
+            "key": "[Ctrl-B] [M-m]"
+        }, {
             "val": "Copy selection",
             "key": "[Ctrl-B] [M-w]"
         }, {
             "val": "Paste selection",
             "key": "[Ctrl-B] [M-y]"
         }, {
+            "val": "Clear selection",
+            "key": "[Ctrl-B] [C-g]"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/tmux.json]: Can you flip this to `[Ctrl-B] [Ctrl-g]` for consistency with the others (Swap panes for example)

Comment (mayank7) [inline share/goodie/cheat_sheets/json/tmux.json]: @mintsoft Sorry, I din't get you. Can you please tell exactly what I need to do.

Comment (moollaza) [inline share/goodie/cheat_sheets/json/tmux.json]: @mayank7 `[C-g]` should be `[Ctrl-g]` to match the others which use Ctrl twice

**Comment by mayank7** [comment]
@moollaza @mintsoft  I made the change suggested by you.

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**